cache/src/commonTest/kotlin/com/dropbox/kmp/external/cache3/TestData.kt

```diff
@@ -0,0 +1,11 @@
+package com.dropbox.kmp.external.cache3
+
+data class TypeWithHashCode(
```

Comment:
nit: please change to non data class as data classes are not binary compatible when making changes https://jakewharton.com/public-api-challenges-in-kotlin/